docker/sui-network/docker-compose.yaml

```diff
@@ -0,0 +1,176 @@
+version: "3"
+
+services:
+  validator1:
+    networks:
+      sui-network:
+        ipv4_address: 10.0.0.11
+    image: mysten/sui-node:mainnet-v1.19.1
```

Comment:
i assume the image tag will become an configurable at some point?

Reply:
As this is just an example for running docker-compose, I'm going to leave the image tag pinned on `:mainnet`, which was reintroduced recently: https://github.com/MystenLabs/sui/pull/16588#discussion_r1525173202